fastboot.jinja2

```diff
@@ -0,0 +1,179 @@
+{% extends "master.jinja2" %}
+
+{% set use_context = true %}
+
+{% if install_packages is undefined %}{% set install_packages = true %}{% endif %}
+{% if install_fastboot is undefined %}{% set install_fastboot = true %}{% endif %}
+{% if ptable is undefined %}{% set prable = false %}{% endif %}
+{% if boot is undefined %}{% set boot = true %}{% endif %}
+{% if boot_label is undefined %}{% set boot_label = "boot" %}{% endif %}
+{% if rootfs is undefined %}{% set rootfs = true %}{% endif %}
+{% if rootfs_label is undefined %}{% set rootfs_label = 'rootfs' %}{% endif %}
+{% if apply_overlay is undefined %}{% set apply_overlay = "rootfs" %}{% endif %}
+{% if deploy_target is undefined %}{% set deploy_target = "fastboot" %}{% endif %}
+{% if pre_power_command is undefined %}{% set pre_power_command = true %}{% endif %}
+{% if pre_os_command is undefined %}{% set pre_os_command = true %}{% endif %}
+{% if auto_login is undefined %}{% set auto_login = true %}{% endif %}
+{% if boot_method is undefined %}{% set boot_method = "fastboot" %}{% endif %}
+
+
+
+{% block context %}
+  test_character_delay: 10
+{% endblock context %}
+
+{% block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-target
+    template: debian
+    distribution: debian
+    release: buster
+    arch: amd64
+    mirror: http://deb.debian.org/debian
+{% endblock protocols %}
+
+{% block actions %}
+{% block lxc %}
+- deploy:
+    namespace: tlxc
+    timeout:
+      minutes: {{ lxc_deploy_timeout }}
+    to: lxc
+    packages:
+    - wget
+    - unzip
+{% if install_packages == true %}
+    - abootimg
+    - cpio
+    - curl
+    - e2fsprogs
+    - file
+    - git
+    - gzip
+    - img2simg
+    - libguestfs-tools
+    - mkbootimg
+    - mktemp
+    - simg2img
+    - xz-utils
+{% endif %}
+    os: debian
+
+- boot:
+    namespace: tlxc
+    prompts:
+    - 'root@(.*):/#'
+    timeout:
+      minutes: {{ lxc_boot_timeout }}
+    method: lxc
+
+{% if install_fastboot == true %}
+- test:
+    namespace: tlxc
+    timeout:
+      minutes: {{ lxc_install_fastboot_timeout }}
+    definitions:
+    - repository: https://github.com/Linaro/test-definitions.git
+      from: git
+      path: automated/linux/android-platform-tools/install.yaml
+      name: install-android-platform-tools-r2600
+      parameters:
+        LINK: 'https://dl.google.com/android/repository/platform-tools_r26.0.0-linux.zip'
```

Comment:
r28 is the latest stable version

Reply:
I'll do that in a separate commit if thats OK